src/postgres/devcontainer-template.json

```diff
@@ -12,6 +12,7 @@
             "description": "Python version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
                 "3",
```

Comment:
Can we bump the patch version here? 👇 
https://github.com/devcontainers/templates/blob/8bcdbe508dc96ed6feb427aad35ed64563085085/src/postgres/devcontainer-template.json#L3

The Template will only be published if the version does not exist, hence needs a version bump.